exercises/binary/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef BINARY_H
+#define BINARY_H
+#define INVALID -1
+
+int convert(char *input);
```

Comment:
Please use `const char *input` instead to clarify that the pointer data won't be modified.